content/en/docs/ops/common-problems/validation/index.md

```diff
@@ -86,54 +86,10 @@ webhooks:
     - '*'
     scope: '*'
   sideEffects: None
-  timeoutSeconds: 30
-{{< /text >}}
-
-If the `validatingwebhookconfiguration` doesn’t exist, verify the
-`istio-validation` `configmap` exists. Istio uses
-the data from this configmap to create and update the
-`validatingwebhookconfiguration`.
-
-{{< text bash yaml >}}
-$ kubectl -n istio-system get configmap istio-validation -o jsonpath='{.data}'
-map[config:apiVersion: admissionregistration.k8s.io/v1beta1
-kind: ValidatingWebhookConfiguration
-metadata:
-  name: istiod-istio-system
-  namespace: istio-system
-  labels:
-    app: istiod
-    release: istio
-    istio: istiod
-webhooks:
-  - name: validation.istio.io
-    clientConfig:
-      service:
-        name: istiod
-        namespace: istio-system
-        path: "/validate"
-        port: 443
-      caBundle: ""
-    rules:
-      - operations:
-        - CREATE
-        - UPDATE
-        apiGroups:
-        - config.istio.io
-        - rbac.istio.io
-        - security.istio.io
-        - authentication.istio.io
-        - networking.istio.io
-        apiVersions:
-        - "*"
-        resources:
-        - "*"
-    failurePolicy: Fail
-    sideEffects: None]
-      (... snip ...)
+  timeoutSeconds: 10
 {{< /text >}}
 
-If the webhook array in `istio-validation` is empty, verify
+If the webhook array in `istio-validator-` is empty, verify
 the `global.configValidation` installation options are
 set.
```

Comment:
Actually, it looks to me like the webhook isn't applied at all if global.configValidation is not set.

```suggestion
If the `istio-validator-` webhook does not exist, verify
the `global.configValidation` installation option is
set to `true`.
```